Bug with DWB rotate to goal critic

I'm seeing this bug in the lifecycle branch, but haven't tested on Master.

Basically the rotate to goal critic is discarding all the proposed trajectories.

[ERROR] [DWBLocalPlanner]: No valid trajectories out of 84! 
[ERROR] [DWBLocalPlanner]: 1.00: RotateToGoal/Nonrotation command near goal.
[ERROR] [dwb_controller]: No valid trajectories out of 84!

--- a/nav2_dwb_controller/nav_2d_utils/include/nav_2d_utils/odom_subscriber.hpp
+++ b/nav2_dwb_controller/nav_2d_utils/include/nav_2d_utils/odom_subscriber.hpp
@@ -70,9 +70,13 @@ public:
         rclcpp::SystemDefaultsQoS(),
         std::bind(&OdomSubscriber::odomCallback, this, std::placeholders::_1));
 
-    nh.get_parameter_or("min_x_velocity_threshold", min_x_velocity_threshold_, 0.0001);
-    nh.get_parameter_or("min_y_velocity_threshold", min_y_velocity_threshold_, 0.0001);
-    nh.get_parameter_or("min_theta_velocity_threshold", min_theta_velocity_threshold_, 0.0001);
+    nh.declare_parameter("min_x_velocity_threshold", rclcpp::ParameterValue(0.0001));
+    nh.declare_parameter("min_y_velocity_threshold", rclcpp::ParameterValue(0.0001));
+    nh.declare_parameter("min_theta_velocity_threshold", rclcpp::ParameterValue(0.0001));
+
+    nh.get_parameter("min_x_velocity_threshold", min_x_velocity_threshold_);
+    nh.get_parameter("min_y_velocity_threshold", min_y_velocity_threshold_);
+    nh.get_parameter("min_theta_velocity_threshold", min_theta_velocity_threshold_);
   }
 
   inline nav_2d_msgs::msg::Twist2D getTwist() {return odom_vel_.velocity;}

